pallets/rolldown/src/lib.rs

```diff
@@ -1078,8 +1124,14 @@ impl<T: Config> RolldownProviderTrait<ChainIdOf<T>, AccountIdOf<T>> for Pallet<T
 			sequencer_set.insert(
 				sequencer.clone(),
 				SequencerRights {
-					read_rights: T::RightsMultiplier::get(),
-					cancel_rights: count.saturating_mul(T::RightsMultiplier::get()),
+					read_rights: if Pallet::<T>::has_read_rights_under_dispute(chain, sequencer) {
+						0u128
```

Comment:
Shouldn't we get the count of the number of read rights under dispute and then sort of `RightsMultiplier - number_of_read_rights_under_dispute`?